src/app/zap-templates/zcl/data-model/chip/namespaces.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+<!--
+TODO: Make these namespace enums global rather than defining them for each cluster.
```

Comment:
should this be removed?

Reply:
I don't think so. I put it there to remind people about the need to do this. Maybe I should add the github issue.

Reply:
Yes, please add the github issue number

Reply:
Added.